applications/job/templates/cronjob.yaml

```diff
@@ -153,6 +153,13 @@ spec:
                 {{- else }}
                 memory: {{ .Values.resources.requests.memory }}
                 {{- end }}
+          {{ if .Values.awsEfsStorage }}
+            volumeMounts:
+          {{- range $v := .Values.awsEfsStorage }}
+            - name: efs-{{ $.Release.Name }}
+              mountPath: /data/efs/{{ $.Release.Name }}
```

Comment:
Can we take out efs from the path here? That would sort of tie us down to this name. We may want to replace with s3, for instance.

Reply:
Sure, took it out on the latest changes, its now set in ccp.